layersvt/monitor_layer.md

```diff
@@ -1,13 +1,18 @@
 <!-- markdownlint-disable MD041 -->
 <p align="left"><img src="https://vulkan.lunarg.com/img/NewLunarGLogoBlack.png" alt="LunarG" width=263 height=113 /></p>
-<p align="left">Copyright � 2015-2021 LunarG, Inc.</p>
+<p align="left">Copyright � 2015-2022 LunarG, Inc.</p>
 
 [![Creative Commons][3]][4]
 
 [3]: https://i.creativecommons.org/l/by-nd/4.0/88x31.png "Creative Commons License"
 [4]: https://creativecommons.org/licenses/by-nd/4.0/
 
 # VK\_LAYER\_LUNARG\_monitor
-The `VK_LAYER_LUNARG_monitor` utility layer prints the real-time frames-per-second value to the application's title bar. It is only compatible with the Win32 and XCB windowing systems and will not produce output on other platforms.
+The `VK_LAYER_LUNARG_monitor` utility layer displays the real-time frame rate in frames-per-second in the application's title bar. It is only compatible with the Win32 and XCB windowing systems and will not display the frame rate on other platforms.
 
-The layer can easily be enabled using the [Vulkan Configurator](https://vulkan.lunarg.com/doc/sdk/latest/windows/vkconfig.html) included with the Vulkan SDK.
+For an overview of how to configure layers, refer to the [Layers Overview and Configuration](https://vulkan.lunarg.com/doc/sdk/latest/windows/layer_configuration.html) document.
+
+The Monitor Layer can be enabled using the [Vulkan Configurator](https://vulkan.lunarg.com/doc/sdk/latest/windows/vkconfig.html) included with the Vulkan SDK.
+## Layer Options
+
+The options for this layer are specified in VK_LAYER_LUNARG_monitor.json. The option details are in [monitor_layer.html](https://vulkan.lunarg.com/doc/sdk/latest/windows/monitor_layer.html#user-content-layer-details).
```

Comment:
Change "The option details are in [monitor_layer.html]" to

"The layer option details are in the [monitor layer documentation]"